mlir/test/python/dialects/transform_structured_ext.py

```diff
@@ -18,33 +19,40 @@ def run(f):
     return f
 
 
+def create_sequence(func: Callable) -> Callable:
+    def decorated() -> None:
+        sequence = transform.SequenceOp(
+            transform.FailurePropagationMode.Propagate,
+            [],
+            transform.AnyOpType.get(),
+        )
+        with InsertionPoint(sequence.body):
+            func(sequence.bodyTarget)
+            transform.YieldOp()
+
+    decorated.__name__ = func.__name__
```

Comment:
`@functools.wraps(func)` on top of `decorated` looks better.

Reply:
Indeed! Fixed with latest commit.